pages/covid.js

```diff
@@ -29,11 +29,15 @@ function COVID() {
           Our plan is approved by <a href="https://sole.ucla.edu/" target="_blank" rel="noopener noreferrer">UCLA SOLE</a> and is a superset of <a href="https://covid-19.ucla.edu/ucla-covid-protocols/" target="_blank" rel="noopener noreferrer">UCLA&apos;s COVID-19 protocols</a> and <a href="http://ph.lacounty.gov/acd/ncorona2019/EducationToolkit/IHE/" target="_blank" rel="noopener noreferrer">LA County Guidelines</a>.
         </p>
         <p>
-          Last major update: <b>Sunday, March 27th, 2022</b>.
+          Last major update: <b>Monday, April 11th, 2022</b>.
           <div>
             <p className='mb-0'>Version history</p>
             <ul className='mt-0'>
-              <li><b>Sunday, March 27th, 2022</b>: updated to reflect <a href='https://covid-19.ucla.edu/ucla-covid-protocols/' target="_blank" rel="noopener noreferrer"> UCLA&apos;s COVID-19 protocol updates for spring quarter</a></li>
+              <li>
+                  <b>Monday, April 11th, 2022</b>
+                  : Updated based on UCLA&apos;s Updates on COVID-19 Protocols for Spring Quarter
```

Comment:
Can you please embed a link into the `UCLA's Updates on COVID-19 Protocols for Spring Quarter` to point to UCLA's covid guidelines like done in the other bullet points? https://covid-19.ucla.edu/update-on-covid-19-protocols-for-spring-quarter/